### PR TITLE
PYIC-1772: Wire up the `evaluate-gpg45-scores` and `select-cri` lambdas with the `process-journey-step` lambda

### PR DESCRIPTION
**Pull request by Danomanic**
### What changed
- Wire up the `evaluate-gpg45-scores` and `select-cri` lambdas with the `process-journey-step` lambda

### Issue tracking

- [PYIC-1772](https://govukverify.atlassian.net/browse/PYIC-1772)

### CODE REVIEW

File: lambdas/process-journey-step/src/main/resources/statemachine/build-statemachine-config.yaml [hunk 1]
```diff
@@ -3,6 +3,13 @@ CRI_STATE:
   name: CRI_STATE
   parent: null
   events:
+    next:
+      type: basic
+      name: next
+      targetState: EVALUATE_GPG45_SCORES
+      response:
+        type: journey
+        journeyStepId: /journey/journey/evaluate-gpg45-scores
```
Comment (Wynndow) [inline lambdas/process-journey-step/src/main/resources/statemachine/build-statemachine-config.yaml]: Should this be just `/journey/evaluate-gpg45-scores`?

File: lambdas/process-journey-step/src/main/resources/statemachine/dev-statemachine-config.yaml [hunk 2]
```diff
@@ -3,6 +3,13 @@ CRI_STATE:
   name: CRI_STATE
   parent: null
   events:
+    next:
+      type: basic
+      name: next
+      targetState: EVALUATE_GPG45_SCORES
+      response:
+        type: journey
+        journeyStepId: /journey/journey/evaluate-gpg45-scores
```
Comment (Wynndow) [inline lambdas/process-journey-step/src/main/resources/statemachine/dev-statemachine-config.yaml]: Same here

File: lambdas/process-journey-step/src/main/resources/statemachine/integration-statemachine-config.yaml [hunk 3]
```diff
@@ -3,6 +3,13 @@ CRI_STATE:
   name: CRI_STATE
   parent: null
   events:
+    next:
+      type: basic
+      name: next
+      targetState: EVALUATE_GPG45_SCORES
+      response:
+        type: journey
+        journeyStepId: /journey/journey/evaluate-gpg45-scores
```
Comment (Wynndow) [inline lambdas/process-journey-step/src/main/resources/statemachine/integration-statemachine-config.yaml]: And here

File: lambdas/process-journey-step/src/main/resources/statemachine/staging-statemachine-config.yaml [hunk 4]
```diff
@@ -3,6 +3,13 @@ CRI_STATE:
   name: CRI_STATE
   parent: null
   events:
+    next:
+      type: basic
+      name: next
+      targetState: EVALUATE_GPG45_SCORES
+      response:
+        type: journey
+        journeyStepId: /journey/journey/evaluate-gpg45-scores
```
Comment (Wynndow) [inline lambdas/process-journey-step/src/main/resources/statemachine/staging-statemachine-config.yaml]: One more

File: lambdas/select-cri/src/main/resources/log4j2.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+Configuration:
```
Comment (Wynndow) [inline lambdas/select-cri/src/main/resources/log4j2.yaml:1]: I think you might be missing the `IpvLamdbaJsonLayout.json` file that goes with this. You can find it in other lambdas Resource dir

**Review by Wynndow — COMMENTED** [review]
Looks good, just a few tweaks to some of the paths.
I also think you need to give the CRI select lambda perms to access config in the param store. So adding to the template.yaml:
```
        - SSMParameterReadPolicy:
            ParameterName: !Sub ${Environment}/core/self/*
```

File: lambdas/select-cri/src/main/resources/log4j2.yaml (hunk 5 above)
Comment (Danomanic) [reply lambdas/select-cri/src/main/resources/log4j2.yaml:1]: Done!

**Review by Wynndow — DISMISSED** [review]
🚀

**Review by Wynndow — APPROVED** [review]

**Pull request merged**